tests/testsuite/cfg.rs

```diff
@@ -817,3 +817,145 @@ fn cfg_booleans_rustflags_no_effect() {
 "#]])
         .run();
 }
+
+#[cargo_test]
+fn cfg_version() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "a"
+                edition = "2015"
+
+                [target.'cfg(version("1.87.0"))'.dependencies]
+                b = { path = 'b' }
+            "#,
+        )
+        .file("src/lib.rs", "extern crate b;")
+        .file("b/Cargo.toml", &basic_manifest("b", "0.0.1"))
+        .file("b/src/lib.rs", "")
+        .build();
+    p.cargo("check -v").run();
+}
```

Comment:
I wish we had a way to override the rust version so we could test with `1.2345` (the version we always use for "far future" in tests)

Reply:
Could we use `RUSTC_FORCE_RUSTC_VERSION`?